docker/app/data/data_augmentation.py

```
@@ -0,0 +1,498 @@
+                logging.error(f"翻訳中にエラーが発生しました: {e}")
+                # エラー発生時は元のテキストをそのまま使用
+                forward_translations.extend(batch)
+
+        # 目標言語 -> 元言語へ逆翻訳
+        batches = [forward_translations[i:i+batch_size] for i in range(0, len(forward_translations), batch_size)]
+        for batch in tqdm(batches, desc="逆翻訳（後方）"):
+            try:
+                # 目標言語から元言語への翻訳
+                translations = self._translate_batch(batch, tgt_lang, src_lang)
+                augmented_texts.extend(translations)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)